v0/26-1-0_Charging_module_–_introduction.markdown

```diff
@@ -1,34 +1,43 @@
-# 26.1.0    Charging module � introduction
+# 26.1.0    Charging module introduction
 
 > The charging module allows you to set charges for attending a {{work}} record, and then decide who should pay that charge. 
 
-The charging module allows you to set charges for attending a {{work}} record, and then decide who should pay that charge. There is a great deal of flexibility in the way that charges are calculated, and the payments due split between different {{people}}. Once these payments have been allocated, they can be batched up into invoices and can then be exported into your own accounting software for billing and credit control purposes. 
+The charging module allows you to set charges for attending a {{work}} record, and then decide who should pay that charge. There is a great deal of flexibility in the way that charges are calculated, and the payments due can be split between different {{people}}. Once these payments have been allocated, they can be batched up into invoices and can then be exported into your own accounting software for billing and credit control purposes. 
 
-The charging module is designed so that it's really quick and easy to use on a day to day basis. This does mean that setting it up is more involved. 
+The charging module is designed so that it's really quick and easy to use on a day to day basis. This does mean that setting it up is much more involved. 
 
 For day-to-day use, when adding a {{work}} record you will see a {{charge}} tab, which lets you choose a charging policy and a payee policy for the record. 
 
+![Charging policy in {{work}} record]({{imgpath}}1222.png)
+
 Charging policies determine how much each {{person}} attending should be charged; Payee policies determine who should pay this charge. 
 
 Both charge and payee policies can take various factors into account in their calculations. You create these policies to do what you need them to do. 
 
-For example, you might have a charging policy that charges everyone £50, and a payee policy that says that the {{person}} attending pays the full amount. A more complex example of a charging policy would: 
+__Creating policies__
+
+For example, you might have a charging policy that charges everyone Â£50, and a payee policy that says that the {{person}} attending pays the full amount. A more complex example of a charging policy would: 
 
  1. Use an hourly rate for the {{workarea}} chosen, a fixed amount for the location used, and the hourly rate for the staff present,
-  2. If {{people}} attending are unemployed, charge them a fixed rate of £10,
+  2. If {{people}} attending are unemployed, charge them a fixed rate of Â£10,
```

Comment:
Oops, can we get rid of the Â encoding issues?!  I'm not sure if this should be &pound; or £ - perhaps keep a single £ and we'll see when it goes live...